src/components/Tabs/components/Tab/Tab.tsx

```diff
@@ -127,7 +127,7 @@ export function Tab({
   );
 
   return (
-    <li className={tabContainerClassNames} ref={node}>
+    <li className={tabContainerClassNames} ref={node} role="tab">
```

Comment:
@jaxee Has a pull request adding `role=presentation` - I think we'll want `role=tab` on the div

Reply:
@AndrewMusgrave see [this thread](https://github.com/Shopify/polaris-react/pull/3653#discussion_r530537915) 😄

Reply:
#3647 has been merged, this can be removed if you rebase atop latest master